src/lib389/lib389/cli_ctl/dblib.py

```diff
@@ -170,7 +258,7 @@ def get_backends(log, dse, tmpdir):
     # now that we finish reading the dse.ldif we may update it if needed.
     for dn, dir in update_dse:
         dse.replace(dn, 'nsslapd-directory', dir)
-    log.debug(f'lib389.cli_ctl.dblib.get_backends returns: {str(res)}')
+    self.log.debug(f'lib389.cli_ctl.dblib.get_backends returns: {str(res)}')
```

Comment:
Probably, it should be just `log.debug(f'lib389...`, as there's no `self` in here.